Dangerfile

```diff
@@ -186,8 +186,10 @@ module VSPDanger
     def run
       required_group = '@department-of-veterans-affairs/backend-review-group'
       exception_groups = %w[@department-of-veterans-affairs/octo-identity
-                            @department-of-veterans-affairs/lighthouse-dash @department-of-veterans-affairs/lighthouse-pivot
-                            @department-of-veterans-affairs/lighthouse-banana-peels]
+                            @department-of-veterans-affairs/lighthouse-dash
+                            @department-of-veterans-affairs/lighthouse-pivot
+                            @department-of-veterans-affairs/lighthouse-banana-peels
+                            @department-of-veterans-affairs/accredited-representatives-admin]
 
```

Comment:
@department-of-veterans-affairs/accredited-representatives-admin is exempt from backend approval